<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enroll</class>
 <widget class="QWidget" name="enroll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,10">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>18</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>18</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">.QFrame{
	background-color: #EDD4D4;
	border-top-left-radius: 20px;
	border-top-right-radius: 20px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>18</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="rightMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="loginBtn">
        <property name="styleSheet">
         <string notr="true">font: 12 14pt &quot;阿里妈妈方圆体 VF Thin&quot;;</string>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>152</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 63 16pt &quot;阿里妈妈方圆体 VF SemiBold&quot;;</string>
        </property>
        <property name="text">
         <string>视频播放器用户注册</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>151</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="styleSheet">
         <string notr="true">font: 12 14pt &quot;阿里妈妈方圆体 VF Thin&quot;;</string>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">.QGroupBox{
	font: 12 14pt &quot;阿里妈妈方圆体 VF Thin&quot;;
	border-top: 2px solid #EDD4D4;
	border-left: 2px solid #EDD4D4;
	border-right: 2px solid #EDD4D4;
}</string>
     </property>
     <property name="title">
      <string>用户注册</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="passwordlabel">
           <property name="styleSheet">
            <string notr="true">font: 57 14pt &quot;阿里妈妈方圆体 VF Medium&quot;;</string>
           </property>
           <property name="text">
            <string>密      码：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="accountlineEdit">
           <property name="minimumSize">
            <size>
             <width>260</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">.QLineEdit{
border-radius: 6px;
border-bottom: 3px solid black;
}</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="enrollBtn">
           <property name="minimumSize">
            <size>
             <width>260</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12 14pt &quot;阿里妈妈方圆体 VF Thin&quot;;

</string>
           </property>
           <property name="text">
            <string>注册</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/new/prefix1/sources_img/登录.png</normaloff>:/new/prefix1/sources_img/登录.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="accountlabel">
           <property name="styleSheet">
            <string notr="true">font: 57 14pt &quot;阿里妈妈方圆体 VF Medium&quot;;</string>
           </property>
           <property name="text">
            <string>账      号：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="repasswordlineEdit">
           <property name="minimumSize">
            <size>
             <width>260</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">.QLineEdit{
border-radius: 6px;
border-bottom: 3px solid black;
}</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="passwordlineEdit">
           <property name="minimumSize">
            <size>
             <width>260</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">.QLineEdit{
border-radius: 6px;
border-bottom: 3px solid black;
}</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="repasswordlabel">
           <property name="styleSheet">
            <string notr="true">font: 57 14pt &quot;阿里妈妈方圆体 VF Medium&quot;;</string>
           </property>
           <property name="text">
            <string>确认密码：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QFrame" name="frame">
           <property name="maximumSize">
            <size>
             <width>761</width>
             <height>50</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="text">
               <string>男</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="text">
               <string>女</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
